README.md

```diff
@@ -21,7 +21,7 @@ so we want to create a list that can help you query the companies much easier. b
 - Backbase [Website](https://www.backbase.com/) - [Linkedin](https://www.linkedin.com/company/backbase/)
 - Flow Traders [Website](https://www.flowtraders.com/) - [Linkedin](https://www.linkedin.com/company/flow-traders/jobs/)
 - Farm21 [Website](https://www.farm21.com/) - [Linkedin](https://www.linkedin.com/company/farm21/)
-- Takeaway.com [Website](https://www.takeaway.com/) - [Linkedin](https://www.linkedin.com/company/just-eat-takeaway-com/)
+- JustEatTakeaway.com [Website](https://careers.justeattakeaway.com/global/en/search-results?keywords=&p=ChIJVXealLU_xkcRja_At0z9AGY) - [Linkedin](https://www.linkedin.com/company/just-eat-takeaway-com/)
```

Comment:
why have you deleted the main takeaway?
now we have 2 takeaway in the list.

Reply:
that website is a user-facing website for ordering food. I replaced that with a corporate website with more information about the company itself and links to the careers page. 

justeat takeaway group has offices in many countries and hires in Germany and the Netherlands.